product_docs/docs/lasso/4/release-notes.mdx

```diff
@@ -10,26 +10,26 @@ Lasso Version 4.13.0 includes the following enhancements and bug fixes:
-| Feature        | Now gathers load profile from `edb_wait_states` | DC-955 |
-| Feature        | Now gathers top wait events from `edb_wait_states` | DC-956 |
-| Feature        | Now gathers top SQL statements from `edb_wait_states` | DC-957 |
-| Feature        | Now gathers transaction stats from `edb_wait_states` | DC-958 |
-| Feature        | Now gathers WAL stats from `edb_wait_states` | DC-959 |
-| Feature        | Now gathers shared buffers stats from `edb_wait_states` | DC-960 |
-| Feature        | Now gathers tuple stats from `edb_wait_states` | DC-961 |
-| Feature        | Now gathers temp files stats from `edb_wait_states` | DC-962 |
-| Feature        | Now gathers information about user sessions from `edb_wait_states` | DC-963 |
-| Feature        | Now gathers information about database settings from `edb_wait_states` | DC-964 |
-| Feature        | Added documentation about `edb_wait_states` into the Lasso docs | DC-972 |
-| Bug fix        | Allow `lasso --version` to be executed without a config file and simplify the output  | RT100700, DC-943 |
-| Bug fix        | No longer fetches `/proc` information if running from inside a container | RT101043, DC-968 |
-| Bug fix        | Added `SYS` schema qualifier for the `ALL_PART_*` relations | RT101043, DC-969 |
+| Feature        | Added Lasso packages for SLES 15. | DC-288 |
+| Feature        | Added new `--info` option to display all collector information with revision/version. | DC-943 |
+| Feature        | Added support for `edb_queue`. | DC-965 |
+| Feature        | Added command-line arguments to specify the range for `edb_wait_states`: `--sampling-start` and `--sampling-end`. | DC-951 |
+| Feature        | Now gather server information from `edb_wait_states`. | DC-953 |
+| Feature        | Now gather operating system information from `edb_wait_states`. | DC-954 |
+| Feature        | Now gather load profile from `edb_wait_states`. | DC-955 |
+| Feature        | Now gather top wait events from `edb_wait_states`. | DC-956 |
+| Feature        | Now gather top SQL statements from `edb_wait_states`. | DC-957 |
+| Feature        | Now gather transaction stats from `edb_wait_states`. | DC-958 |
+| Feature        | Now gather WAL stats from `edb_wait_states`. | DC-959 |
+| Feature        | Now gather shared buffers stats from `edb_wait_states`. | DC-960 |
+| Feature        | Now gather tuple stats from `edb_wait_states`. | DC-961 |
+| Feature        | Now gather temp files stats from `edb_wait_states`. | DC-962 |
+| Feature        | Now gather information about user sessions from `edb_wait_states`. | DC-963 |
+| Feature        | Now gather information about database settings from `edb_wait_states`. | DC-964 |
+| Feature        | Added documentation about `edb_wait_states` to the Lasso docs | DC-972 |
+| Bug fix        | `lasso --version` can now be executed without a config file and the output was simplified.  | RT100700, DC-943 |
```

Comment:
Check this, I wasn't sure if this was what was meant.

Reply:
@lucianobotti , what do you think?